### ISSUE TITLE
Missing react-a11y converter: react-a11y-lang

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#902]
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 2.0.1
-   `typescript-eslint` version: 4.9.1

#### TSLint Rule Name

`react-a11y-lang`

Per https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md, the equivalent rules are now `jsx-a11y/html-has-lang` and `jsx-a11y/lang`.

#### Package Source

https://github.com/teamairship/tslint-react-a11y
